cockatrice/src/dialogs/dlg_settings.cpp

```diff
@@ -1251,13 +1251,24 @@ void SoundSettingsPage::retranslateUi()
 
 ShortcutSettingsPage::ShortcutSettingsPage()
 {
+    // search bar
+    searchEdit = new SearchLineEdit;
+    searchEdit->setObjectName("searchEdit");
+    searchEdit->setPlaceholderText(tr("Search by shortcut name"));
```

Comment:
translateable text should go in retranslateUI

Reply:
Oops… feel free to open a PR to fix it

Reply:
Fixing